_includes/themes/bootstrap/post.html

```diff
@@ -97,7 +110,7 @@ <h4>Tags</h4>
                         {% endfor %}
                     </div>
 
-                    <div class="mb-25"></div>
+                    <!-- <div class="mb-25"></div>
```

Comment:
Let's just remove this instead of commenting it out - it's in git history if we need to bring it back

Reply:
done